Sources/Common/Core/Math/index.js

```diff
@@ -45,25 +44,21 @@ const { round, floor, ceil, min, max } = Math;
 
 function arrayMin(arr) {
   let minValue = Infinity;
-  for (let i = 0, len = arr.length; i < len; i += MAX_FUNCTION_ARGUMENTS) {
-    const submin = Math.min.apply(
-      null,
-      arr.slice(i, Math.min(i + MAX_FUNCTION_ARGUMENTS, len))
-    );
-    minValue = Math.min(submin, minValue);
+  for (let i = 0, len = arr.length; i < len; ++i) {
+    if (arr[i] < minValue) {
+      minValue = arr[i];
+    }
   }
 
   return minValue;
 }
 
 function arrayMax(arr) {
   let maxValue = -Infinity;
-  for (let i = 0, len = arr.length; i < len; i += MAX_FUNCTION_ARGUMENTS) {
-    const submax = Math.max.apply(
-      null,
-      arr.slice(i, Math.min(i + MAX_FUNCTION_ARGUMENTS, len))
-    );
-    maxValue = Math.max(submax, maxValue);
+  for (let i = 0, len = arr.length; i < len; ++i) {
+    if (arr[i] > maxValue) {
```

Comment:
you should use `<` as `>` is interpreted differently in JS.

Reply:
I'll bite. How is > interpreted?

Reply:
```
Note: The algorithm is only defined for a < b. So, a > b is handled as b < a.
```

https://github.com/getify/You-Dont-Know-JS/blob/master/types%20%26%20grammar/ch4.md

I guess it doesn't matter since the Browser will reverse it...
But in case the reverse action take time? ;-)

Reply:
Thanks for the tip @jourdain . JavaScript is sometimes `<` optimal, but we'll go with it :-P. Adding in a follow-up commit.